components/brave_shields/browser/brave_shields_util.h

```diff
@@ -29,6 +29,11 @@ namespace brave_shields {
 
 enum ControlType { ALLOW = 0, BLOCK, BLOCK_THIRD_PARTY, DEFAULT, INVALID };
 
+struct CookiesState {
```

Comment:
I don't understand the reason for this struct. We already have too many different ways to represent cookie settings and adding another one doesn't seem necessary. Also we have yet another one here https://github.com/brave/brave-core/pull/14449/files#diff-4adc2a8e53a85a82220e33b2bb1292390cd78de042d264c5b33438a61ed5b1b5R89